src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -2451,27 +2451,6 @@ internal override Symbol RemapSymbolIfNecessaryCore(Symbol symbol)
                     {
                         switch (declaredSymbol.Kind)
                         {
-                            case SymbolKind.Method:
```

Comment:
>case SymbolKind.Method: [](http://example.com/codeflow?start=28&length=23)

The change feels incomplete. It feels like this should become unreachable, i.e. more changes should be made elsewhere. I'd recommend taking a look at the PR that added this case and see other related changes.

Reply:
Thanks. For now, I'm going to leave that for @mavasani. I can try with it later if @mavasani didn't get the time to finish it.